app/controllers/hyrax/batch_edits_controller.rb

```diff
@@ -39,7 +39,15 @@ def destroy_collection
         obj = ActiveFedora::Base.find(doc_id, cast: true)
         obj.destroy
       end
-      flash[:notice] = "Batch delete complete"
+
+      admin_set_name = "Administrative Set"
+      admin_set_name = AdminSet.first.title.first unless AdminSet.first.nil?
```

Comment:
Of all the admin sets, why are you choosing the first one? Why are you testing for nil?

Reply:
Changing it to: `admin_set_type_name = Hyrax::CollectionType.find_or_create_admin_set_type.title`